### PR TITLE
Add cilium bump version test

### PR DESCRIPTION
**Pull request by bguzman-3pillar**
Adding `Cilium` version bump test.

- Check the cilium version
- Network checks

### CODE REVIEW

**Review by est-suse — COMMENTED** [review]
can you please tell me how to run this?  Just an example

File: entrypoint/versionbump/cilium_test.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+			assert.PodAssertReady(),
+			assert.PodAssertStatus())
+	})
+
+	It("Verifies bump version on rke2 for cilium version", func() {
+		template.VersionTemplate(template.VersionTestTemplate{
+			TestCombination: &template.RunCmd{
+				Run: []template.TestMap{
+					{
+						Cmd: "sudo /var/lib/rancher/rke2/bin/crictl --config /var/lib/rancher/rke2/agent/etc/crictl.yaml images | grep cilium ," +
```
Comment (fmoral2) [inline entrypoint/versionbump/cilium_test.go:40]: break the line sooner to not exced limit....break after yaml.

File: entrypoint/versionbump/cilium_test.go [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+			TestConfig: &template.TestConfig{
+				TestFunc:       template.ConvertToTestCase(customflag.ServiceFlag.TestConfig.TestFuncs),
+				DeployWorkload: customflag.ServiceFlag.TestConfig.DeployWorkload,
+				WorkloadName:   customflag.ServiceFlag.TestConfig.WorkloadName,
+			},
+			Description: customflag.ServiceFlag.TestConfig.Description,
+		})
+	})
+
+    It("Verifies ClusterIP Service", func() {
```
Comment (fmoral2) [inline entrypoint/versionbump/cilium_test.go:57]: do we want always run these tests bellow ?

Comment (bguzman-3pillar) [reply entrypoint/versionbump/cilium_test.go:57]: Yes, we usually do network checks after validate new cni versions

File: entrypoint/versionbump/cilium_test.go (hunk 1 above)
Comment (bguzman-3pillar) [reply entrypoint/versionbump/cilium_test.go:40]: Fixed

**Comment by bguzman-3pillar** [comment]
> can you please tell me how to run this? Just an example

Sure, should be like:
```
# Just for 1 version/commit
go test -timeout=45m -v -tags=cilium ./entrypoint/versionbump/... -expectedValue "v1.14.0, v1.25" 

# Using upgrade
go test -timeout=45m -v -tags=cilium ./entrypoint/versionbump/... -expectedValue "v1.14.0, v1.25" -expectedValueUpgrade "v1.14.0, v1.26" -installVersionOrCommit <commit> -channel testing
```

**Review by fmoral2 — APPROVED** [review]

**Review by est-suse — APPROVED** [review]

**Pull request merged**